src/app/style/styles.js

```diff
@@ -0,0 +1,67 @@
+export default {
+  header: {
+    color: "#1297A1",
+    fontFamily: "Roboto Condensed",
+  },
+  body: {
+    padding: "0 100px",
+    flex: "3",
+  },
+  nav: {
+    height: "60px",
+    padding: "20px 100px",
+  },
+  navItem: {
+    display: "inline-block",
+  },
+  navList: {
+    listStyle: "none",
+    paddingLeft: "0",
+    margin: "0",
+    display: "flex",
+  },
+  navLink: {
+    color: "#3EBEC7",
```

Comment:
This color `#3ebec7` against white has a contrast ratio of 2.24, but at
the size at which it appears the minimum contrast ratio is 4.5. Could
you please increase the contrast to meet the accessibility guidelines?
(tota11y suggests using `#00848b`.)

Reply:
Sad, I liked that teal.